### DISCUSSION TITLE
Flexdasboard (.Rmd) in a quarto website or blog

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#2562] [Q&A] [answered]
Dear list members,

Is it posible to deploy a flexdashboard (.Rmd) in a quarto website or blog?

**Comment by mcanouil** [comment]
In a weird not easy, not native, and not in a well/properly integrated way.
You probably better wait for the "flexdashboard" feature to arrive.

If you really want to do it, I see the following way:
1. Render your Rmd file to html
2. Embed in a Quarto document using iframe.

**Comment by ManuelSpinola** [comment]
Thank you very much Mickaël.

Any idea when the flexdashboard feature will be available?

Manuel



El jue, 22 sept 2022 a las 12:47, Mickaël Canouil ***@***.***>)
escribió:

> In a weird not easy, not native, and not in a well/properly integrated way.
> You probably better wait for the "flexdashboard" feature to arrive.
>
> If you really want to do it, I see the following way:
>
>    1. Render your Rmd file to html
>    2. Embed in a Quarto document using iframe.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2562#discussioncomment-3711964>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBZIKZPMCYQL25XSRFLV7SSTLANCNFSM6AAAAAAQTHGKHQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Reply by cderv** [reply, marked as answer]
There is no ETA on that but it will definitely be a new format we will work on.

**Comment by mcanouil** [comment]
Duplicate of #843
